admin/class-import-wpseo.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * @package WPSEO\Admin\Import\External
+ */
+
+/**
+ * Class WPSEO_Import_WPSEO
+ *
+ * Class with functionality to import Yoast SEO settings from wpSEO
+ */
+class WPSEO_Import_WPSEO extends WPSEO_Import_External {
+
+	/**
+	 * @var array
+	 */
+	private $robot_values = array(
+		1 => array( 'index' => 2, 'follow' => 0 ),
+		2 => array( 'index' => 2, 'follow' => 1 ),
+		3 => array( 'index' => 1, 'follow' => 0 ),
+		4 => array( 'index' => 1, 'follow' => 0 ),
+		5 => array( 'index' => 1, 'follow' => 1 ),
+		6 => array( 'index' => 2, 'follow' => 0 ),
```

Comment:
The value of key 6 is the same as the value of key 1. Is there a reason for that?